src/scenes/home/benefit/benefit.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Section from 'shared/components/section/section';
+import LinkButton from 'shared/components/linkButton/linkButton';
+import styles from './benefit.css';
+
+const Benefit = () => (
+  <div>
+    <Section title="Operation Code Veteran’s Day Benefit Dinner & Silent Auction" theme="white">
```

Comment:
This title is too long and will break our <Heading> component on many viewports.